src/parser/json/relationship_parser.py

```diff
@@ -0,0 +1,182 @@
+                try:
+                    contains_relationship = Relationship(spdx_element_id=package_spdx_id,
+                                                         relationship_type=RelationshipType.CONTAINS,
+                                                         related_spdx_element_id=file_spdx_id)
+                except ConstructorTypeErrors as err:
+                    logger.append(err.get_messages())
+                    continue
+                if not self.check_if_relationship_exists(relationship=contains_relationship,
+                                                         existing_relationships=existing_relationships):
+                    contains_relationships.append(contains_relationship)
+        raise_parsing_error_if_logger_has_messages(logger, "package contains relationships")
+
+        return contains_relationships
+
+    def check_if_relationship_exists(self, relationship: Relationship,
+                                     existing_relationships: List[Relationship]) -> bool:
+        existing_relationships_without_comments: List[Relationship] = self.get_all_relationships_without_comments(
+            existing_relationships)
+        if relationship in existing_relationships_without_comments:
+            return True
+        relationship_inverted: Relationship = self.invert_relationship(relationship)
+        if relationship_inverted in existing_relationships_without_comments:
+            return True
+
+        return False
+
+    @staticmethod
+    def get_all_relationships_without_comments(existing_relationships: List[Relationship]) -> List[Relationship]:
+        relationships_without_comments = [Relationship(relationship_type=relationship.relationship_type,
+                                                       related_spdx_element_id=relationship.related_spdx_element_id,
+                                                       spdx_element_id=relationship.spdx_element_id) for relationship in
+                                          existing_relationships]
+        return relationships_without_comments
+
+    def invert_relationship(self, relationship: Relationship) -> Relationship:
+        return Relationship(related_spdx_element_id=relationship.spdx_element_id,
+                            spdx_element_id=relationship.related_spdx_element_id,
+                            relationship_type=self.invvert_relationship_types[relationship.relationship_type],
+                            comment=relationship.comment)
+
+    invvert_relationship_types = {RelationshipType.DESCRIBES: RelationshipType.DESCRIBED_BY,
+                                  RelationshipType.DESCRIBED_BY: RelationshipType.DESCRIBES,
+                                  RelationshipType.CONTAINS: RelationshipType.CONTAINED_BY,
+                                  RelationshipType.CONTAINED_BY: RelationshipType.CONTAINS}
+
+    @staticmethod
+    def parse_file_dependencies(file_dicts: List[Dict]) -> List[Relationship]:
```

Comment:
shouldn't there also be a `check_if_relationship_exists` in this method?

Reply:
as discussed in a call I deleted the formed code and opened an issue to implement this once it is clear how this field is converted